.gitignore

```diff
@@ -1,7 +1,9 @@
 target
 .idea
-
+.bloop
 .DS_Store
-
+.vscode
+.metals
 # you should probably un-ignore this in your own project
 yarn.lock
+node_modules
```

Comment:
I don't think `node_modules` is needed here because Scala.js puts these inside `target`, which is already ignored.

Reply:
I've removed this on the fork, apparently, this PR is supposed to update itself...